testsuite/tests/compaction/test_quick_compact.ml

```diff
@@ -0,0 +1,41 @@
+let keep_alive = ref []
+
+let generate_garbage () =
+  keep_alive := [];
+  (* start with a very similar heap state *)
+  begin
+    Gc.full_major ();
+    Gc.compact ();
+    Gc.minor ()
```

Comment:
If I understand correctly, calling `full_major` before `compact` is redundant. (I also wonder why you call `Gc.minor()` after that. I would rather call it before `compact` to ensure that all live blocks are on the major hear?)

Reply:
You are right, there's no need to call `full_major`. This test originally comes from when `compact` didn't do a full major first.

The call to `minor` is to ensure that quick_stat reflects the right stats - after we made major update stats we no longer need that either.